integration_test/third_party_apps_data/applications/dcgm/metadata.yaml

```diff
@@ -23,28 +23,35 @@ configure_integration: |-
   You must install DCGM and run the DCGM daemon service. 
 supported_operating_systems: linux
 supported_app_version: ["3.1"]
-platforms_to_skip:
-  - centos-7
-  - centos-8
-  - debian-10
-  - debian-11
+platforms_to_skip: #skipping arm64 platforms, and those without supported driver
```

Comment:
`platforms_to_skip` feels redundant with `gpu_platforms`; can we just omit it and use `gpu_platforms` to decide which platforms to run the tests on?